test/gmp.jl

```diff
@@ -468,6 +468,34 @@ end
     end
 end
 
+@testset "modular invert" begin
+    # test invert is correct and does not mutate
+    a = BigInt(3)
+    b = BigInt(7)
+    i = BigInt(5)
+    @test Base.GMP.MPZ.invert(a, b) == i
+    @test a == BigInt(3)
+    @test b == BigInt(7)
+
+    # test in place invert does mutate first argument
+    a = BigInt(3)
+    b = BigInt(7)
+    i = BigInt(5)
+    i_inplace = BigInt(3)
+    Base.GMP.MPZ.invert!(i_inplace, b)
+    @test i_inplace == i
+
+    # test in place invert does mutate only first argument
+    a = BigInt(3)
+    b = BigInt(7)
+    i = BigInt(5)
+    i_inplace = BigInt(0)
+    Base.GMP.MPZ.invert(i_inplace, a, b) == i
```

Comment:
Was this supposed to be
```suggestion
    Base.GMP.MPZ.invert!(i_inplace, a, b) == i
```
?

Reply:
Yes. And the comparison was also redundant, sorry :D I did not know, how to run Julia lang tests yet, now I know

Reply:
[`Base.runtests("gmp")`](https://docs.julialang.org/en/v1/stdlib/Test/#Base.runtests)